aperturedb/NotebookHelpers.py

```diff
@@ -60,7 +86,7 @@ def draw_bboxes(image, boxes=[], tags=[], save=False):
         cv2.imwrite(img_file, cv_image)
 
 
-def display_video_mp4(blob):
+def display_video_mp4(blob, show=True):
```

Comment:
I think "display_video" should just display it. We can have another method that will only write videos to files, to avoid overloading this method.

Reply:
ok will address.